packages/web-components/fast-foundation/src/utilities/property-stylesheet-behavior.ts

```diff
@@ -0,0 +1,71 @@
+import {
+    Behavior,
+    ElementStyles,
+    FASTElement,
+    Observable,
+} from "@microsoft/fast-element";
+
+/**
+ * A behavior to add or remove a stylesheet from an element based on a property. The behavior ensures that
+ * styles are applied while the property matches and that styles are not applied if the property does
+ * not match.
+ *
+ * @public
+ */
+export class PropertyStyleSheetBehavior implements Behavior {
+    /**
+     * Constructs a {@link PropertyStyleSheetBehavior} instance.
+     * @param propertyName - The property name to operate from.
+     * @param value - The property value to operate from.
+     * @param styles - The styles to coordinate with the property.
+     */
+    constructor(
+        private propertyName: string,
+        private value: any,
+        private styles: ElementStyles
+    ) {}
+
+    /**
+     * Binds the behavior to the element.
+     * @param elementInstance - The element for which the property is applied.
+     */
+    public bind(elementInstance: FASTElement) {
+        Observable.getNotifier(elementInstance).subscribe(this, this.propertyName);
```

Comment:
We should probably also check the value at this point and add the styles if appropriate, otherwise we're only adding the styles if the value changes from its initial value.

Reply:
The easiest way to do this is to just call `handleChange` directly.

Reply:
@eljefe223 you need to call the function w/ the appropriate arguments:

```
this.handleChange(elementInstance, this.propertyName);
```